.moban.d/travis.yml

```diff
@@ -13,5 +12,7 @@ python:
   - 3.7
   - 3.6
   - 3.5
+  - 3.4
```

Comment:
The reason why 3.4 was wiped off is because pypi-mobans has removed 3.4 in the upstream hence any update would have removed 3.4 without notice. 

And then, to remove 3.4 was my naive thought that not many people uses 3.4 and 3.4 is not as good as python 3.5.